static/docs/commands-reference/remote:add.md

```diff
@@ -41,6 +42,13 @@ and optionally assigns a default remote in the core section:
    remote = myremote
 ```
 
+DVC supports the concept of a _default remote_.  For the commands which take
+a `--remote` option, this option can be left off the command line and the
+default remote will be used instead.
+
+Use `dvc config` to unset/change the default remote as so:
+`dvc config -u core.remote`.
```

Comment:
minor, you can split code blocks: `test
test`

Reply:
sorry, it put in on single line, but basically you can start a code block in one line and complete on the next one )

Reply:
However in my editor the syntax coloring breaks, and as a person reading the Markdown it becomes harder to track